src/cirrus/builtins/functions/update-state/lambda_function.py

```diff
@@ -75,13 +73,13 @@ def from_event(cls, event: Dict[str, Any]) -> "Execution":
             status = event["detail"]["status"]
             error = None
 
-            if status == SUCCEEDED:
+            if status == SfnStatus.SUCCEEDED:
```

Comment:
if only we could use pattern matching :)

Reply:
so close...we could early drop py3.9 ;)